armi/materials/tests/test_uraniumOxide.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tests for Uranium Oxide.
+
+Note that more tests for UO2 are in tests/test_materials.py
+"""
+
+import unittest
+
+from armi.materials.uraniumOxide import UO2
+from armi.materials.tests.test_materials import _Material_Test
+
+
+class UraniumOxide_TestCase(_Material_Test, unittest.TestCase):
+    MAT_CLASS = UO2
+
+    def setUp(self):
+        _Material_Test.setUp(self)
+
+    def test_applyInputParams(self):
+        # TODO add other matMods
```

Comment:
We try not to add new `TODO` in a PR.  We should add more tests or remove the `TODO`, in an ideal world.

Reply:
Done!